src/esaml.erl

```diff
@@ -645,4 +656,25 @@ validate_stale_assertion_test() ->
     }),
     {error, stale_assertion} = validate_assertion(E1, "foobar", "foo").
 
+decode_id_metadata_with_multiple_bindings_test() ->
```

Comment:
Minor: should these be named with "idp" instead of "id".. eg, "decode_idp_metadata_with_multiple_bindings_text"